bindings/pydeck/pydeck/bindings/map_styles.py

```diff
@@ -1,4 +1,60 @@
-SATELLITE = "mapbox://styles/mapbox/satellite-v9"
-LIGHT = "mapbox://styles/mapbox/light-v9"
-DARK = "mapbox://styles/mapbox/dark-v9"
-DARK_NO_LABELS = "https://rivulet-zhang.github.io/dataRepo/mapbox/style/map-style-dark-v9-no-labels.json"
+import warnings
+
+
+DARK = "dark"
+LIGHT = "light"
+SATELLITE = "satellite"
+ROAD = "road"
+DARK_NO_LABELS = "dark_no_labels"
+LIGHT_NO_LABELS = "light_no_labels"
+
+MAPBOX_LIGHT = "mapbox://styles/mapbox/light-v9"
+MAPBOX_DARK = "mapbox://styles/mapbox/dark-v9"
+MAPBOX_DARK_NO_LABELS = "https://rivulet-zhang.github.io/dataRepo/mapbox/style/map-style-dark-v9-no-labels.json"
```

Comment:
Why are you depending on a style hosted on someone else's site? What if they delete it?